File: 프로그래머스/0/120837. 개미 군단/개미 군단.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int hp) {
    int answer = 0;
    int ant=1;
    int sant=3;
    int cant=5;
    int count=(hp/cant)+((hp%cant)/sant)+(((hp%cant)%sant)/ant);
    answer=count;
    return answer;
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.js
function solution(hp) {
    var answer = 0;
    let ant=1;
    let sant=3;
    let cant=5;
    let count=Math.floor(hp/cant)+Math.floor((hp%cant)/sant)+Math.floor(((hp%cant)%sant)/ant);
    answer=count;
    return answer;
}